RSConnectDIOToSnap7/CodeBehind.cs

```diff
@@ -36,203 +36,205 @@
 
 namespace RSConnectDIOToSnap7
 {
-	/// <summary>

```

Comment:
Indent was tab characters...

Reply:
what is the point with the space vs tab chars? it has simply no difference for me. it reads the same and code works also the same.

Reply:
This is for code review.
I prefer use tab for indent as each one can change its size (2-4 or 8 chars).